net-im/toxic/toxic-0.8.2.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1 xdg
+
+DESCRIPTION="A curses-based client for Tox"
+HOMEPAGE="https://github.com/JFreegman/toxic"
+SRC_URI="https://github.com/JFreegman/toxic/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/fx-carton/toxic_patches/archive/v${PV}.tar.gz -> ${PN}_patches-${PV}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+X +audio notifications +python +qrcode +video"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+# Not a typo; net-libs/tox only has a 'both or neither' option
+RDEPEND="
+	audio? ( net-libs/tox:0/0.2[av] )
+	video? ( net-libs/tox:0/0.2[av] )
+	net-libs/tox:0/0.2
+	dev-libs/libconfig
+	net-misc/curl:0=
+	sys-libs/ncurses:0=
+	audio? (
+		media-libs/openal
+		media-libs/freealut
+	)
+	video? (
+		media-libs/libvpx:=
+		x11-libs/libX11
+	)
+	notifications? ( x11-libs/libnotify )
+	python? ( ${PYTHON_DEPS} )
+	qrcode? ( media-gfx/qrencode:= )
+"
+DEPEND="
+	virtual/pkgconfig
+	${RDEPEND}
+"
+
+PATCHES=(
+	"${WORKDIR}/${PN}_patches-${PV}/${P}-make-qrencode-optional.patch"
+	)
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+	if [[ ${PV} = 0.8.2 ]]; then
+		sed -i \
+			-e 's/^\(TOXIC_VERSION =\).*$/\1 0.8.2/' \
+			cfg/global_vars.mk || die
+	else
+		die "Wrong version"
```

Comment:
Please use helpful error messages. 'Wrong version' tells me nothing. 'Remove stale version hack' would be much clearer.